### PR TITLE
Assignment Part 1

### PR DESCRIPTION
**Pull request by RobertElliottKrieter**
I hope this worked-- sorry for the trouble!

### CODE REVIEW

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 1]
```diff
@@ -1,145 +1,114 @@
...
+For example, John (x) is connected to Mary (y) by object-property "has father" is unique.
+That is, Steve (y) can only bear the object-property relationship "has_father" with one other individual, namely, John (x).
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+When running a reasoner the combination of these becomes undecideable because a functional property must only be only be obtained between one individual and another,
+however, if it is applied transitively it whill carry over to subsequent pairs who are also connected by this same object property.
```
Comment (elenamiliv) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html]: typo: will instead of will

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 2]
```diff
@@ -1,145 +1,114 @@
...
+
+Transitive Property Assertion: brother_of
+Irreflexive Property assertion: brother_of 
+
+    x is brother of y
+    y is brother of z
+    z is brother of x
+Therefore, x is brother to itself. 
+
+This leads to a contradiction because x cannot be brother to itself by virtue of irreflexivity.
```
Comment (elenamiliv) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html]: Let R be a transitive property and irreflexive. Transitivity implies that if xRy and yRz, then xRz. Irreflexivity, on the other hand, prohibits any reflexive relationships (i.e., xRx is forbidden). In a scenario where the transitive property causes x to relate to itself (e.g., xRy, yRz, xRz), this creates a contradiction with irreflexivity, leading to undecidability

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 3]
```diff
@@ -1,145 +1,114 @@
...
+6.) ASYMMETRICAL + REFLEXIVE = UNSATISFIABLE
+
+Asymmetry provides that if (x) is related to (y) by object_property_3 then (y) CANNOT relate to (x) by object_property_3
+Ex: Asymmetric_Object_Property (Parent_of)
+    Object_Property_Assertion (parent_of, Vivian, Chris)  = "Vivian is a parent of Chris"
+    It holds that Chris CANNOT be a parent of Vivian. 
+
+Reflexivity holds that an instance can relate to itself by some object property, (aRa) where a is some instance. 
+
+If (x) relates to some (y) and why, in turn, this (y) is able to point back to (x) by virtue of (x)'s reflexivity then this would lead to a contradiction.
```
Comment (elenamiliv) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html]: Reflexivity implies that xRx for any instance x. Asymmetry, however, requires that if xRy, then yRx is prohibited. This creates a direct contradiction, as reflexivity mandates self-relations, which asymmetry forbids. Therefore, the combination is unsatisfiable.

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 4]
```diff
@@ -1,145 +1,114 @@
...
+If (x) relates to some (y) and why, in turn, this (y) is able to point back to (x) by virtue of (x)'s reflexivity then this would lead to a contradiction. 
+
+
+7.) REFLEXIVE + IRREFLEVITY = UNSATISFIABLE
+
+Irreflexive hold that no individual can be connected by such an object property to itself.
+
+Reflexivity holds that an instance can relate to itself by some object property, (aRa) where a is some instance.
+
+If some instance (x) has object property that relates to itself concatenated with another that is mutually exclusive of itself then this will
```
Comment (elenamiliv) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html]: Reflexive properties require xRx for all instances x, while irreflexive properties prohibit xRx. These two constraints are directly contradictory, leading to unsatisfiability in any case where a property is declared both reflexive and irreflexive

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 5]
```diff
@@ -1,145 +1,114 @@
...
+Ex: Symmtric_Object_Property (sister)
+    Object_Property_Assertion (sister, Mary, Jen)  = "Mary is sister of Jen"
+    It follows that "Jen is sister of Mary" because the relation is symmetrically held between both instances. 
+
+Asymmetry provides that if (x) is related to (y) by object_property_3 then (y) CANNOT relate to (x) by object_property_3
+Ex: Asymmetric_Object_Property (Parent_of)
+    Object_Property_Assertion (parent_of, Vivian, Chris)  = "Vivian is a parent of Chris"
+    It holds that Chris CANNOT be a parent of Vivian. 
+
+The joint of these leads to unsatisfiability because an object property cannot relate mutually between instances while also only applying one direction between one to the other.
```
Comment (elenamiliv) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html]: Let R be an object property that is both symmetric and asymmetric. Symmetry implies that for any xRy, it must hold that yRx. However, asymmetry requires that if xRy, then it cannot be the case that yRx. These two conditions are logically contradictory, resulting in unsatisfiability

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 6]
```diff
@@ -1,145 +1,114 @@
...
+When running a reasoner the combination of these becomes undecideable because a functional property must only be only be obtained between one individual and another,
+however, if it is applied transitively it whill carry over to subsequent pairs who are also connected by this same object property.
+
+
+2.) INVERSE-FUNCTIONALITY AND TRANSITIVITY BECOMES UNDECIDEABLE
+
+Inverse functionality provides that each individual paired with another with this relation can have at most one incoming connection of said object property.
+Ex: Inverse_Functional_Property "father of";
+    Father_of (Joey, Richard)      = "Joey is father_of Richard"
+--> This means that the instance, Richard, can only be inverse-fucntionally related to Joey and no other instance because any person can only have one father.
```
Comment (elenamiliv) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html]: typo: functionally

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 7]
```diff
@@ -1,145 +1,114 @@
...
+--> Conversely the object-property "father_of" may be obtained between Joey and other instances, who will only have this relation which Joey. 
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+When running a reasoner the combination of these becomes undecideable because a functional property must only be only be obtained between one individual and another,
```
Comment (elenamiliv) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html]: I would rephrase this: an inverse functional property ensures that an object can be connected to at most one subject through a given object property

**Comment by RobertElliottKrieter** [comment]
Thank you!!

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html (hunk 2 above)
Comment (johnbeve) [reply Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html]: @elenamiliv Almost, but I wouldn't say the contradiction results in undecidability. You can leave it at contradiction ;)
